pkg/system/phase1_verifying.go

```diff
@@ -252,3 +273,26 @@ func (r *Reconciler) checkExternalPg(postgresDbURL string) error {
 	}
 	return nil
 }
+
+func (r *Reconciler) checkBucketLoggingPVC() error {
+	BucketLoggingPVC := &corev1.PersistentVolumeClaim{
+		TypeMeta: metav1.TypeMeta{Kind: "PersistenVolumeClaim"},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      *r.NooBaa.Spec.BucketLogging.BucketLoggingPVC,
+			Namespace: r.Request.Namespace,
+		},
+	}
+	if !util.KubeCheck(BucketLoggingPVC) {
+		return util.NewPersistentError("InvalidBucketLoggingConfiguration",
+			"'Guaranteed' BucketLogging requires a Persistent Volume Claim (PVC) with ReadWriteMany (RWX) access mode. The specified 'BucketLoggingPVC' was not found")
+	}
+
+	// Check if pvc supports RWX access mode
+	for _, accessMode := range BucketLoggingPVC.Spec.AccessModes {
+		if accessMode == corev1.ReadWriteMany {
+			return nil
+		}
+	}
+	return util.NewPersistentError("InvalidBucketLoggingConfiguration",
+		"'Guaranteed' BucketLogging requires a Persistent Volume Claim (PVC) with ReadWriteMany (RWX) access mode. The specified 'BucketLoggingPVC' does not support RWX access mode")
```

Comment:
Also here we can take this out:
```suggestion
		"The specified 'BucketLoggingPVC' does not support RWX access mode")
```